pydis_site/apps/content/tests/helpers.py

```diff
@@ -75,29 +75,32 @@ class MockPagesTestCase(fake_filesystem_unittest.TestCase):
 
     def setUp(self):
         """Create the fake filesystem."""
-        self.setUpPyfakefs()
-
-        self.fs.create_file(f"{BASE_PATH}/_info.yml", contents=CATEGORY_INFO)
-        self.fs.create_file(f"{BASE_PATH}/root.md", contents=MARKDOWN_WITH_METADATA)
-        self.fs.create_file(
-            f"{BASE_PATH}/root_without_metadata.md", contents=MARKDOWN_WITHOUT_METADATA
-        )
-        self.fs.create_file(f"{BASE_PATH}/not_a_page.md/_info.yml", contents=CATEGORY_INFO)
-        self.fs.create_file(f"{BASE_PATH}/category/_info.yml", contents=CATEGORY_INFO)
-        self.fs.create_file(
-            f"{BASE_PATH}/category/with_metadata.md", contents=MARKDOWN_WITH_METADATA
-        )
-        self.fs.create_file(f"{BASE_PATH}/category/subcategory/_info.yml", contents=CATEGORY_INFO)
-        self.fs.create_file(
-            f"{BASE_PATH}/category/subcategory/with_metadata.md", contents=MARKDOWN_WITH_METADATA
-        )
-        self.fs.create_file(
-            f"{BASE_PATH}/category/subcategory/without_metadata.md",
-            contents=MARKDOWN_WITHOUT_METADATA
-        )
+        Path(f"{BASE_PATH}/_info.yml").write_text(CATEGORY_INFO)
```

Comment:
```suggestion
        (BASE_PATH / "_info.yml").write_text(CATEGORY_INFO)
```
potentially neater to let pathlib handle putting these together, although not sure it actually makes a different so should be fine as is.